src/lib/migration-parser.ts

```diff
@@ -60,7 +94,12 @@ const createMigrationParser = function (fetcher): (migrationCreator: (migration:
 
     const ctsWithEntryInfo = await fetcher.checkContentTypesForDeletedCts(intentList, contentTypes)
 
-    validateChunks(intentList, ctsWithEntryInfo)
+    const payloadValidationErrors = validateChunks(intentList, ctsWithEntryInfo)
```

Comment:
nit (also not change on this PR), it feels a bit strange that on line 60 we do `intentList.validate()` and here we pass the `intentList` to this function for further validation.